shut-down.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+docker-compose down --remove-orphans
+cd external_tools/
+docker-compose down
+cd ..
+docker volume rm external_tools_cassandra-volume
```

Comment:
since you are destroying all data I would rename this file something like "shut-down-and-reset.sh"